backend/internal/system/crypto/encrypt/service.go

```diff
@@ -16,8 +16,8 @@
  * under the License.
  */
 
-// Package crypto provides cryptographic functionality with algorithm agility.
-package crypto
+// Package encrypt provides cryptographic functionality with algorithm agility.
+package encrypt
```

Comment:
With the package restructuring, should this `CryptoService` renamed to `EncryptionService`. Suggesting this because it only handles encryption/ decryption. 
Can be addressed with a separate effort